src/nbl/ext/ImGui/shaders/common.hlsl

```diff
@@ -9,19 +9,48 @@
 	struct PSInput
```

Comment:
`VSInput` shouldn't be in common

Also ever struct should be in `nbl::ext::imgui` namespace

Reply:
(maybe `emulated_snorm16_t2` can be just in `nbl::hlsl`)